UserManual.md

```diff
@@ -0,0 +1,49 @@
+# QueryForge
+**User Manual**
+
+## Introduction
+QueryForge web application helps students and researchers find suitable research projects and faculty mentors within the ECE department. By gathering information from university websites and lab pages, the tool provides a one-stop platform for exploring research opportunities based on user-defined interests. The app also offers a resume parser that matches user skills and keywords with relevant projects.
+
+---
+
+## Accessing the Application
+1. **Open the Web Application**: Go to the application URL (http://3.142.98.164:5000/) in your preferred web browser.
+2. **Login**: Sign in using your credentials. If you are new to the the application, you will be asked to create an account.
+---
+
+## Features and Usage
+
+### Home Page
+- After logging in, the **Home Page** displays a search bar, filters, and current highlights of research opportunities within the university.
+- Users can search directly for professors, projects, or keywords to see available opportunities.
+
+### Searching for Research Opportunities
```

Comment:
Also lets refer to our Functionality page as SEARCH page not HOME page. I think our HOME page just says QueryForge.

Reply:
Good point! Fixed.